lib/datasource/pod/index.ts

```diff
@@ -98,26 +119,54 @@ const githubRegex = regEx(
   /^https:\/\/github\.com\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
 );
 
+const githubEnterpriseRegex = regEx(
+  /(?<hostURL>(^https:\/\/github\..+\.[a-zA-z0-9-]+))\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
+);
+
 async function getReleasesFromGithub(
   lookupName: string,
-  registryUrl: string,
-  useShard = false
+  opts: { hostURL: string; account: string; repo: string },
+  useShard = true,
+  useSpecs = true,
+  urtFormatOptions = URLFormatOptions.WithShardWithSpec
```

Comment:
```suggestion
  urlFormatOptions = URLFormatOptions.WithShardWithSpec
```

Reply:
thanks for finding the typo. Updated in 3594dd917.